satpy/dataset/dataid.py

```diff
@@ -254,7 +254,9 @@ def __hash__(self):
                                   "brightness_temperature",
                                   "radiance",
                                   "radiance_wavenumber",
-                                  "counts"
+                                  "counts",
+                                  "aerosol_thickness",
+                                  "water_vapor"
```

Comment:
I understand that this is needed, but instead of adding it globally, I recommend making a dataid just for this reader, see https://github.com/pytroll/satpy/blob/main/satpy/etc/readers/sgli_l1b.yaml#L13-L31
More generally, it feels a bit wrong to have aeorosol thickness as a calibration type, but we have no way of having composites included directly here, so this will have to do. (@djhoese any comment on this?)

Reply:
That's what I've been looking for...

Reply:
I have a question: If I don't change it in dataid, how can I call these two in test codes?

Reply:
@djhoese Can you help me with this?

Reply:
Ok I think I found the way. I just need to override `make_dataid`